styles.css

```diff
@@ -0,0 +1,1336 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap");
+
+@media only screen and (min-width: 767px) {

```

Comment:
Kindly ensure that the CSS styling is built as mobile-first, then you use the media queries to change the design from mobile to desktop and not the other way round. In order to achieve this kindly remove the `@media only screen and (min-width: 767px) {}` since that is for desktop. That should come after you have stated all the styling for the mobile version. :confused: